packages/sveltekit/src/server/handle.ts

```diff
@@ -1,192 +1,6 @@
-    // if we should create a new execution context or not.
-    // In case of a same-origin `fetch` call within a server`load` function,
-    // SvelteKit will actually just re-enter the `handle` function and set `isSubRequest`
-    // to `true` so that no additional network call is made.
-    // We want the `http.server` span of that nested call to be a child span of the
-    // currently active span instead of a new root span to correctly reflect this
-    // behavior.
-    // As a fallback for Kit < 1.21.0, we check if there is an active span only if there's none,
-    // we create a new execution context.
-    const isSubRequest = typeof input.event.isSubRequest === 'boolean' ? input.event.isSubRequest : !!getActiveSpan();
-
-    if (isSubRequest) {
-      return instrumentHandle(input, options);
-    }
-
-    return withIsolationScope(isolationScope => {
-      // We only call continueTrace in the initial top level request to avoid
-      // creating a new root span for the sub request.
-      isolationScope.setSDKProcessingMetadata({
-        normalizedRequest: winterCGRequestToRequestData(input.event.request.clone()),
-      });
-      return continueTrace(getTracePropagationData(input.event), () => instrumentHandle(input, options));
-    });
-  };
-
-  return sentryRequestHandler;
-}
-
-async function instrumentHandle(
-  { event, resolve }: Parameters<Handle>[0],
-  options: SentryHandleOptions,
-): Promise<Response> {
-  if (!event.route?.id && !options.handleUnknownRoutes) {
-    return resolve(event);
-  }
-
-  const routeName = `${event.request.method} ${event.route?.id || event.url.pathname}`;
-
-  if (getIsolationScope() !== getDefaultIsolationScope()) {
-    getIsolationScope().setTransactionName(routeName);
-  } else {
-    DEBUG_BUILD && logger.warn('Isolation scope is default isolation scope - skipping setting transactionName');
-  }
-
-  try {
-    const resolveResult = await startSpan(
-      {
-        op: 'http.server',
-        attributes: {
-          [SEMANTIC_ATTRIBUTE_SENTRY_ORIGIN]: 'auto.http.sveltekit',
-          [SEMANTIC_ATTRIBUTE_SENTRY_SOURCE]: event.route?.id ? 'route' : 'url',
-          'http.method': event.request.method,
-        },
-        name: routeName,
-      },
-      async (span?: Span) => {
-        getCurrentScope().setSDKProcessingMetadata({
-          normalizedRequest: winterCGRequestToRequestData(event.request.clone()),
-        });
-        const res = await resolve(event, {
-          transformPageChunk: addSentryCodeToPage(options),
-        });
-        if (span) {
-          setHttpStatus(span, res.status);
-        }
-        return res;
-      },
-    );
-    return resolveResult;
-  } catch (e: unknown) {
-    sendErrorToSentry(e, 'handle');
-    throw e;
-  } finally {
-    await flushIfServerless();
-  }
+/** Documented in `worker/handle.ts` */
+export function initCloudflareSentryHandle(_options: any): Handle {
```

Comment:
while `any` isn't wrong here, we have a lint rule against `any` usage. let's just use unknown here as it doesn't matter

```suggestion
export function initCloudflareSentryHandle(_options: unknown): Handle {
```